openslides_backend/action/actions/motion/reset_state.py

```diff
@@ -70,7 +71,6 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
             FullQualifiedId(Collection("motion_state"), instance["state_id"]),
             ["set_created_timestamp"],
         )
-        instance["created"] = None
-        if state.get("set_created_timestamp"):
+        if not motion.get("created") and state.get("set_created_timestamp"):
```

Comment:
Only fetch the state if needed, i.e. if `created` is not set

Reply:
Done.